src/FormUser.js

```diff
@@ -0,0 +1,152 @@
+
+    _handleChangeFirstName(event){
+        this.setState({
+            textn: event.target.value
+        });
+    }
+
+    _handleChangeLastName(event){
+        this.setState({
+            textl: event.target.value
+        });
+    }
+
+    _handleChangeCountry(event){
+        this.setState({
+            textc: event.target.value
+        });
+    }
+
+    _handleChangeAddress(event){
+        this.setState({
+            texta: event.target.value
+        });
+    }
+
+    _handleChangeEmail(event){
+        this.setState({
+            texte: event.target.value
+        });
+    }
+
+    _handleChangePhoneNumber(event){
+        this.setState({
+            textp: event.target.value
+        });
+    }
+
+    _handleClick(){
+        let newUser = {
+            id : Date.now(),
+            firtsname : this.state.textn,
+            lastname: this.state.textl,
+            country: this.state.textc,
+            address: this.state.texta,
+            email: this.state.texte,
+            phonenumber: this.state.textp
+        }
+        
+        this.setState(prevState => (
+            {
+                users: prevState.users.concat(newUser),
+                textn: '',
+                textl: '',
+                textc: '',
+                texta: '', 
+                texte: '',
+                textp: ''
+            }
+        ));    
+    }
+
+    render(){
+        return(
+            <div>  
+                <label>Firts Name: </label>
+                <input 
+                    onChange = {this._handleChangeFirstName}
+                    value = {this.state.textn}>
+                </input>
+
+                <label>Last Name: </label>
+                <input 
+                    onChange = {this._handleChangeLastName}
+                    value = {this.state.textl}>
+                </input>
+
+                <label>Country :</label>
+                <input
+                    onChange = {this._handleChangeCountry}
+                    value = {this.state.textc}>
+                </input>
+
+                <label>Address :</label>
+                <input
+                    onChange = {this._handleChangeAddress}
+                    value = {this.state.texta}>
+                </input>
+
+                <label>Email: </label>
+                <input
+                    onChange = {this._handleChangeEmail}
+                    value = {this.state.texte}>
+                </input>
+
+                <label>Phone Number: </label>
+                <input
+                    onChange = {this._handleChangePhoneNumber}
+                    value = {this.state.textp}>
+                </input>
+
+                <button className='save' onClick = {this._handleClick}>Save!!</button>
+
+                <ul>
+                    {this.state.users.map(user => (
+                        <div className='container'>
+                            <div key={user.id} className='prueba'>
+                                <table>
```

Comment:
Table is not recommended for this kind of 'containers', please refactor this using <div>. If it's necessary, add boostrap 4.

Reply:
Done.